lib/features/home_view/widgets/science_clubs_section.dart

```diff
@@ -23,6 +25,13 @@ class ScienceClubsSection extends ConsumerWidget {
             actionTitle: context.localize.list,
             onClick: ref.navigateScienceClubs,
           ),
+          FilledButton(
+            onPressed: () {
+              unawaited(ref.navigateBuildingGuideDetail("1"));
+            },
+            //onPressed: ref.navigateGuide,
+            child: const Text("Navigate to digital guide screen!"),
```

Comment:
If it is temporary you can comment it. If not, this string shouldn't be hardcoded